libraries/BLE/src/ArduinoBLE.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
In library.property, the library should be have version as 2.0,  the folder name remains as CurieBLE.